### ISSUE TITLE
Subnet

### ISSUE THREAD

**Issue by StarTech42** [tsujamin/hass-addons#4]
Many thanks for this, it's a really useful addon. I'm surprised there's not been more interest. Worked first time for me and I'm a complete beginner. If you are still maintaining it...   

There are a few updates now to the latest version of tailscale. I think they are at about 1.6 as of today.
Would it be possible to enable the subnet routes and relay nodes on the hass version?
The tailscale ip needed :8123 before it could find my HA. I didn't see it in the documentation, but may have missed it.
Are HA allowing submissions to the repository yet, there might be more interest if more users could see the addon?

Thanks again...   it's working great as it is!

**Comment by tsujamin** [comment]
I'll try take a look at this over the weekend 👍

**Comment by tsujamin** [comment]
`advertise-route` got merged in to 1.6.0.0 during the week. I've got a `1.6.0.1` under test that will add exit node support in case you want to run it as a full VPN. I also added some instructions regarding the port 8123 omission you mentioned so other's don't get stuck (thanks 😄). Should hopefully have it out this afternoon or tomorrow depending on how long it takes me to rebuild my hassos today.

Not sure regarding the HA main addon repository - I'll keep an ear out when they re-open. Honestly this has more users than I ever expected!
